modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -426,6 +429,9 @@
 			return columnId;
 		},
 
+		/**
+		 * @deprecated As of Cavanaugh (7.4.x), with no direct replacement
+		 */
```

Comment:
We should say here that the util is replaced by `openModal`.

Reply:
@markocikos we might have overlooked something here. `getWindow()` returns a YUI node-ish object:

<img width="465" alt="image" src="https://user-images.githubusercontent.com/24564752/164706088-5f54b6b8-3399-45d1-8b41-eede40e4f753.png">

whereas `openModal` [renders a modal](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js#L336)

This has been brought up by my talk with @magjed4289 where she discovered that I haven't removed/replaced any of the usages of `getWindow()`

My question to you is: has `getWindow` really been replaced by `openModal`? You're more familiar with the whole modal infrastructure and were the one to comment about the replacement, so just wanna get your quick thought before I dig deeper into this.

Reply:
After investigating this further, it seems to have been a hasty deprecation and seems that the relation between `getWindow` and `openModal` is non-exitant.

So, as it stands, `getWindow` invokes the `Window` variant of `getWindow` with an `id` that is registered by [Window._register](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util_window.js#L405) method.

Now, the question is:
Do we just move `getWindow` to `frontend-js-web`, since there's nothing to modernize
or
Do we keep the deprecation and update the usages of `Liferay.Util.getWindow` to `Util.getTop().Liferay.Util.Window.getById(id)`

There are 43 usages (some are probably inside deprecated aui modules)

I vote for moving to `frontend-js-web` to avoid the potential of dozens of regressions and uglying up the codebase.

Thoughts @markocikos @bryceosterhaus ?

Reply:
>My question to you is: has getWindow really been replaced by openModal

@kresimir-coko  Well, yes and no. We replaced it in a sense that `openModal` is the main util we are using and promoting for developing modal windows. Product teams should be able to implement any modal solution with `openModal`. If we are deprecating `getWindow`, we should state that was replaced by `openModal`. That's what I meant in the review.

But, did we do the migration? I don't think so. Should we deprecate in those circumstances? Probably not. It would be a slippery slope, it's much cleaner to deprecate only unused utils.

Reply:
> I vote for moving to frontend-js-web to avoid the potential of dozens of regressions and uglying up the codebase.

Yeah this makes sense to me as well